lib/Test/Nginx/Socket.pm

```diff
@@ -2372,6 +2396,14 @@ to run the specified number of duplicate requests for each test block. When it i
 
 Default to 1.
 
+=head2 stop_after_request
+
+By default, the nginx is not still running after the request. The error log is incompletely, missing logs after nginx stoped.
```

Comment:
I'm not sure about this feature. Since by default, nginx is quit after each test block is run, unless you set the environment `TEST_NGINX_NO_CLEAN=1`. So we do not really need this section IMHO.